src/app/common/components/Status/Status.styles.ts

```diff
@@ -1,22 +1,27 @@
 import { makeStyles } from '@material-ui/core/styles';
 
 export const useStyles = makeStyles(() => ({
-    OperatorActive: {
+    Big: {
+        height: 28,
+        fontSize: 14,
+        borderRadius: 2,
+        fontWeight: 500,
+        textAlign: 'center',
+        width: 'fit-content',
+        padding: '5px !important',
+    },
+    Normal: {
         height: 20,
         fontSize: 12,
-        // margin: 0 103px 0 12px;
         fontWeight: 600,
         width: 'fit-content',
-        padding: '2px 5px 3px 4px',
+    },
+    Active: {
```

Comment:
Better approach would be:

```
// Somewhere in common/consts ..
enum ActivenessStatus {
  ACTIVE = 'Active',
  INACTIVE = 'Inactive',
  NO_VALIDATORS = 'No validators',
}
```

In this file:

```
import { ActivenessStatus } from '...';
...
[ActivenessStatus.ACTIVE]: {...styles...},
[ActivenessStatus.INACTIVE]: {...styles...},
...
```

And then look at comment in `src/app/common/components/Status/Status.tsx` here in PR